### PR TITLE
New package: Hashicorp.Terraform.Beta version 1.6.0-beta1

### PR DESCRIPTION
**Pull request by kdpuvvadi**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/119274)

### CODE REVIEW

**Comment by kdpuvvadi** [comment]
What seems to be the problem?, working fine on machine and on the sandbox test.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335141 
image

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 2023-09-19 12:47:20.588 [CLI ] Extracting archive to: C:\Users\User\AppData\Local\Temp\WinGet\Hashicorp.Terraform.Beta.1.6.0-beta1\extracted  > 2023-09-19 12:47:21.761 [CLI ] Successfully extracted archive  > 2023-09-19 12:47:21.764 [CLI ] Unable to locate nested installer at: C:\Users\User\AppData\Local\Temp\WinGet\Hashicorp.Terraform.Beta.1.6.0-beta1\extracted\a  > 2023-09-19 12:47:21.764 [CLI ] Terminating context: 0x8a15005b at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\ArchiveFlow.cpp:62

File: manifests/h/Hashicorp/Terraform/Beta/1.6.0-beta1/Hashicorp.Terraform.Beta.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+InstallerType: zip
+FileExtensions:
+- tf
+ReleaseDate: 2023-08-31
+Installers:
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: ./terraform.exe
+  - RelativeFilePath: a
```
Comment (stephengillie) [inline manifests/h/Hashicorp/Terraform/Beta/1.6.0-beta1/Hashicorp.Terraform.Beta.installer.yaml:16]:
```suggestion
```
Hi @kdpuvvadi,

I think this extra entry might be causing the validation error.

**Comment by kdpuvvadi** [comment]
How the hell did i miss that?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**